src/main/java/core/basesyntax/IsoscelesTrapezoid.java

```diff
@@ -0,0 +1,29 @@
+package core.basesyntax;
+
+public class IsoscelesTrapezoid extends Figure {
+    private int upbothside;
+    private int downbothside;
+    private int height;
```

Comment:
Variable names should be descriptive and follow camelCase naming conventions. Consider renaming 'upbothside' to 'upperSide' and 'downbothside' to 'lowerSide' for better readability.